storybook/src/docs/components/Toolbar/stories/FeedbackButton.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import { FeedbackButton, Toolbar, ToolbarActions, ToolbarFillSpace, ToolbarTitleItem } from "@comet/admin";
+import { Assets } from "@comet/admin-icons";
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+import { FormattedMessage } from "react-intl";
+
+import { storyRouterDecorator } from "../../../../story-router.decorator";
+import { toolbarDecorator } from "../toolbar.decorator";
+
+storiesOf("stories/components/Toolbar/Feedback Button", module)
+    .addDecorator(toolbarDecorator())
+    .addDecorator(storyRouterDecorator())
+    .add("Feedback", () => {
+        const [loading, setLoading] = React.useState(false);
+        return (
+            <Toolbar>
+                <ToolbarTitleItem>Feedback Button</ToolbarTitleItem>
+                <ToolbarFillSpace />
+                <ToolbarActions>
+                    <FeedbackButton
+                        color="primary"
+                        variant="contained"
+                        loading={loading}
+                        onClick={() => {
+                            setLoading(true);
+                            setTimeout(() => {
+                                setLoading(false);
+                            }, 1000);
+                        }}
+                        startIcon={<Assets />}
+                        tooltipSuccessMessage="Saving was successful"
+                        tooltipErrorMessage="Error while saving"
+                    >
+                        <FormattedMessage id="comet.feedback" defaultMessage="Feedback" />
```

Comment:
AFAIK we don't use `FormattedMessage` in stories, as it keeps the code simpler and there is no need to translate them.

Reply:
In some stories they are used and in others not. But I get the point, so I deleted it here.